BMW/bmw/page-3.html

```diff
@@ -11,7 +11,7 @@
       <div class="page-1">
         <div class="title-1">
           <h2 style="color: white">
-            <font size="7"> BMW M4</font>
+            <font size="7"> BMW M4 <span id="BMW M4 G82"></span></font>
```

Comment:
do not need this span

Reply:
> Fix all files with prettier. Config it so that 1 tab equals 2 spaces

All files are already fixed with prettier and tab is equal to 2 spaces .But in some files Prettier worked incorrectly and nothing changed